Builds failing on Windows due to pathing issues

__Describe the Bug__

On Windows specifically, the package cannot be built at all, as multiple modules relating to preact cannot be resolved.

```
ERROR in ./preact/hooks/dist/hooks.module.js 1:0-37
Module not found: Error: Can't resolve '....' in 'C:\dev\bpmn-io\properties-panel\properties-panel\preact\hooks\dist'
 @ ./src/PropertiesPanel.js 1:0-54 84:30-38 85:2-11
 @ ./test/spec/PropertiesPanel.spec.js 5:0-50 258:21-36
 @ ./test/ sync .spec\.js$ ./spec/PropertiesPanel.spec.js test/spec/PropertiesPanel.spec.js
 @ ./test/testBundle.js 1:17-57
 ```

__Steps to Reproduce__

1. Clone the repo on a Windows machine
2. Execute `npm install`
3. Execute `npm run dev`


__Expected Behavior__

We expect the build to complete without errors.

__Environment__

- Host: Node v14
- OS: Window 10
- Library version: 0.10.0

--- a/.github/workflows/CI.yml
+++ b/.github/workflows/CI.yml
@@ -5,7 +5,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [ ubuntu-latest ]
+        os: [ macos-latest, ubuntu-latest, windows-latest ]
         node-version: [ 14 ]
 
     runs-on: ${{ matrix.os }}
@@ -29,8 +29,16 @@ jobs:
     - name: Install dependencies
       run: npm ci
     - name: Build
+      if: matrix.os == 'ubuntu-latest'
       env:
         COVERAGE: 1
+        TEST_BROWSERS: ChromeHeadless
+      run: npm run all
+    - name: Build
+      if: matrix.os != 'ubuntu-latest'
+      env:
+        TEST_BROWSERS: ChromeHeadless
       run: npm run all
     - name: Upload coverage
+      if: matrix.os == 'ubuntu-latest'
       run: npx codecov

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -1,4 +1,4 @@
-import { parse as parsePath, relative as relativePath } from 'path';
+import path, { parse as parsePath, relative as relativePath } from 'path';
 import { replaceInFile } from 'replace-in-file';
 
 import babel from '@rollup/plugin-babel';
@@ -64,9 +64,8 @@ function rewirePreactSubpackages() {
                 filePath = args.pop();
 
           const { dir } = parsePath(filePath);
-          const pathToPreact = relativePath(dir, './preact');
-
-          return `${importGroup}${pathToPreact}${afterImport}`;
+          const posixPathToPreact = relativePath(dir, './preact').split(path.sep).join(path.posix.sep);
+          return `${importGroup}${posixPathToPreact}${afterImport}`;
         }
       });
     }

